### DISCUSSION TITLE
How to hide a topic/ask Monika to not bring a topic up again? HELP

### DISCUSSION THREAD

**Discussion by EColi07** [Monika-After-Story/MonikaModDev#9899] [QA - tech support + general]
Title, I forgot what the hot key is and it isn't listed in the settings nor the full list on the official website.

**Comment by lunulae** [comment]
Press X while the topic is in progress, and she'll ask at the end of it.

**Reply by EColi07** [reply]
Thank you, but is it normal for this to not work with some topics? When I press X for certain topics nothing shows up, Monika doesn't ask me for topic removal at the end either.

**Reply by Booplicate** [reply]
What topic are you trying to hide?

**Reply by EColi07** [reply]
The "Natsuki's letter" one

**Reply by Booplicate** [reply]
You don't need to hide that topic as Monika will never bring it up again unless you ask her to.

**Reply by EColi07** [reply]
Got it, thank you!
